README.md

```diff
@@ -1,2 +1,257 @@
+julia> fm = FOOOF(; peak_width_limits=py"[1,8]"o)
+PyObject <fooof.objs.fit.FOOOF object at 0x7fea38b5b040>
+```
+
+Another conversion problem arises in cases where nesting lists and eltypes creates problems.
+For example, the [FOOOF tutorial "Tuning and Troubleshooting"](https://fooof-tools.github.io/fooof/auto_tutorials/plot_07-TroubleShooting.html) includes this statement
+```python
+>>> gauss_params = [[10, 1.0, 2.5], [20, 0.8, 2], [32, 0.6, 1]]
+```
+When executed via PyCall, this results in a Julia `Matrix` with eltype `Real`.
+
+```julia
+julia> py"$([[10, 1.0, 2.5], [20, 0.8, 2], [32, 0.6, 1]])"
+PyObject [array([10. ,  1. ,  2.5]), array([20. ,  0.8,  2. ]), array([32. ,  0.6,  1. ])]
+julia> py"$([[10, 1.0, 2.5], [20, 0.8, 2], [32, 0.6, 1]])"
+3-element Vector{Vector{Float64}}:
+ [10.0, 1.0, 2.5]
+ [20.0, 0.8, 2.0]
+ [32.0, 0.6, 1.0]
+```
+This results in an error:
+```julia
+julia> gen_power_spectrum = fooof.sim.gen.gen_power_spectrum;
+
+julia> f_range = [1, 50];
+
+julia> ap_params = [20, 2];
+
+julia> nlv = 0.1;
+
+julia> gauss_params = [[10, 1.0, 2.5], [20, 0.8, 2], [32, 0.6, 1]];
+
+julia> freqs, spectrum = gen_power_spectrum(f_range, ap_params, gauss_params, nlv)
+ERROR: PyError ($(Expr(:escape, :(ccall(#= /home/ubuntu/.julia/packages/PyCall/BD546/src/pyfncall.jl:43 =# @pysym(:PyObject_Call), PyPtr, (PyPtr, PyPtr, PyPtr), o, pyargsptr, kw))))) <class 'ValueError'>
+ValueError('operands could not be broadcast together with shapes (99,) (3,) ')
+...
+```
+
+When the statement is executed in Python (`py"[]"`) and then roundtripped through Julia, PyCall converts the Python return value to a `Matrix`, which works in the subsequent function call:
+```julia
+julia> gauss_params = py"[[10, 1.0, 2.5], [20, 0.8, 2], [32, 0.6, 1]]"
+3×3 Matrix{Real}:
+ 10  1.0  2.5
+ 20  0.8  2
+ 32  0.6  1
+julia> freqs, spectrum = gen_power_spectrum(f_range, ap_params, gauss_params, nlv)
+([1.0, 1.5, 2.0, 2.5, 3.0, 3.5, 4.0, 4.5, 5.0, 5.5  …  45.5, 46.0, 46.5, 47.0, 47.5, 48.0, 48.5, 49.0, 49.5, 50.0], [9.112713501760112e19, 6.707288550822094e19, 3.4304395055235047e19, 1.6048034860916263e19, 1.3121468876633584e19, 9.23648446980319e18, 7.068034503219047e18, 7.474675398285033e18, 5.682794734823231e18, 6.002884162025267e18  …  5.494028369147603e16, 5.044411758605143e16, 4.528833513498138e16, 4.080554951080287e16, 4.064069219484658e16, 3.9731296024126536e16, 3.21719026879766e16, 4.828351597256686e16, 4.441592192173848e16, 4.129641670786365e16])
+```
+
+However, naively using a true 2d array (`Matrix`) in Julia also results in error:
+```julia
+julia> gauss_params = [10 1.0 2.5; 20 0.8 2; 32 0.6 1]
+3×3 Matrix{Float64}:
+ 10.0  1.0  2.5
+ 20.0  0.8  2.0
+ 32.0  0.6  1.0
+julia> freqs, spectrum = gen_power_spectrum(f_range, ap_params, gauss_params, nlv)
+\ERROR: PyError ($(Expr(:escape, :(ccall(#= /home/ubuntu/.julia/packages/PyCall/BD546/src/pyfncall.jl:43 =# @pysym(:PyObject_Call), PyPtr, (PyPtr, PyPtr, PyPtr), o, pyargsptr, kw))))) <class 'ValueError'>
```

Comment:
Does the error message here actually start with a backslash? And is the line below missing a closing )?